content/docs/apis.md

```diff
@@ -2,57 +2,176 @@
 title: APIs
 description: Interact with TiKV using the raw key-value API or the transactional key-value API
 weight: 4
-draft: true
+draft: false
 ---
 
 TiKV offers two APIs that you can use to interact with it:
 
-* A [raw](#raw) API
-* A [transactional](#transactional) API
+API | Description | Atomicity | Use when...
+:---|:------------|:----------|:-----------
+[Raw](#raw) | A lower-level key-value API for interacting directly with individual key-value pairs. | Single key | Your application doesn't require distributed transactions or multi-version concurrency control (MVCC)
+[Transactional](#transactional) | A higher-level key-value API that provides ACID semantics | Multiple keys | Your application requires distributed transactions and/or MVCC
+
+{{< info >}}
+It is **not recommended or supported** to use both the raw and transactional APIs on the same keyspace.
+{{< /info >}}
+There are several clients that connect to TiKV:
+
+* [Rust](https://github.com/tikv/client-rust)
+* [Java](https://github.com/tikv/client-java)
+* [Go](https://github.com/pingcap/tidb/store/tikv)
+
+Below we use the Rust client for some examples, but you should find all clients work similarly.
+
+## Connecting a Client
+
+This guide assumes you are using Rust and have Rust installed through `rustup` or other means.
+
+To start, open the `Cargo.toml` of your project and add the `tikv-client` as a dependency. You'll also need to depend on `futures`.
```

Comment:
Is there no sync API? We should fix that if not :-)

Reply:
There is not! A sync API not just introduce a `.wait()` internally instead?

Reply:
Maybe, we might be able to do slightly better than that by 'batching'

Reply:
Maybe! You're encouraged to write an RFC about this. But this isn't a blocker on this PR. I'll resolve this.